pkgs/by-name/op/openvas-smb/package.nix

```diff
@@ -0,0 +1,64 @@
+{ fetchFromGitHub
+, lib
+, stdenv
+, cmake
+, git
+, pkg-config
+, glib
+, gnutls
+, perl
+, heimdal
+, popt
+, libunistring
+}:
+stdenv.mkDerivation rec {
+  pname = "openvas-smb";
+  version = "22.5.6";
+
+  src = fetchFromGitHub {
+    owner = "greenbone";
+    repo = "openvas-smb";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-wnlBOHYOTWNbwzoHCpsXbuhp0uH3wBH6+Oo4Y+zSsfg=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    git
+    pkg-config
+  ];
+
+  buildInputs = [
+    glib
+    gnutls
+    perl
+    heimdal
+    popt
+    libunistring
+  ];
+
+  patches = [
+    # add default config header for heimdal https://github.com/heimdal/heimdal/blob/master/include/heim_threads.h
+    ./0001-add-heim_threads.h.patch
+  ];
```

Comment:
We can just copy the whole file, which makes diff's a lot easier to understand or copy that from the heimdal source.

Reply:
Thanks for the feedback! Changed it to use fetchurl and cp. "nixpkgs-review pr 308588" still succeeds.